package.json

```diff
@@ -51,29 +51,30 @@
   ],
   "dependencies": {
     "@cesium/engine": "^6.1.0",
-    "@cesium/widgets": "^4.3.0"
+    "@cesium/widgets": "^4.3.0",
+    "meshoptimizer": "^0.20.0"
```

Comment:
I don't think this needs to be in the top-level `package.json`, only in `packages/engine/package.json`.

Reply:
Thanks for catching that. This should be good to go now